atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/UnfilteredCassandraClientInstrumentation.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.keyvalue.cassandra;
+
+import com.codahale.metrics.MetricRegistry;
+import com.google.common.collect.ImmutableMap;
+import com.palantir.atlasdb.logging.LoggingArgs;
+import com.palantir.tritium.metrics.registry.MetricName;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
+
+public final class UnfilteredCassandraClientInstrumentation implements CassandraClientInstrumentation {
```

Comment:
Once we add the "top list filtered" implementation, we can use this only for tests.

It's also possible to add a config flag to choose which impl in BTG scenarios. Usually, I'd use the logger debug flag but that's more of a runtime config and it's quite awkward to use a runtime-as-install flag. It's a bit counterintuitive to novel readers of this code path / surprising.

In the later PR, I'd either not include the option to fall back or add an AtlasDbConfig flag as a fail safe here. Or, at least something that would adjust the top list size.